### ISSUE TITLE
annotation.on.supertype and instanceof.nullable are not reported for aliased annotations

### ISSUE THREAD

**Issue by aosen-xiong** [eisop/checker-framework#2021]
The Nullness Checker reports `annotation.on.supertype` and `instanceof.nullable` for its own
`@Nullable`, but not for `org.jspecify.annotations.Nullable`, which is an alias for it.

```java
import org.checkerframework.checker.nullness.qual.Nullable;

class Base {}

public class Alias extends @Nullable Base {
    void m(Object o) {
        boolean b = o instanceof @Nullable String;
    }
}
```

```
$ javac -processor nullness Alias.java
Alias.java:5: error: [annotation.on.supertype] do not write type annotations on supertypes in a class declaration
public class Alias extends @Nullable Base {
                           ^
Alias.java:7: error: [instanceof.nullable] instanceof is only true for a non-null expression
        boolean b = o instanceof @Nullable String;
                      ^
2 errors
```

Changing only the import, to `org.jspecify.annotations.Nullable`, produces no output.

Expected: the same two diagnostics, since the annotation is an alias.

Actual: no diagnostic.

`annotation.on.supertype` is issued from `BaseTypeVisitor`, so it applies to every checker that has
aliased qualifiers, not only the Nullness Checker.

Reproduced on master at `e434ef1b4e`.
